packages/cli/src/util/create-git-meta.ts

```diff
@@ -54,6 +54,14 @@ export async function createGitMeta(
   const dirty = dirtyResult.value;
   const commit = commitResult.value;
 
+  let ciMetadata: CiMetadata = {};
+  if (process.env.GITHUB_ACTIONS && process.env.GITHUB_ACTOR) {
+    ciMetadata = {
+      ciType: 'github-actions' as const,
+      githubActionsActor: process.env.GITHUB_ACTOR,
```

Comment:
Not that familiar with this part of the code, but isn't the git metadata usually generic here? Wouldn't we want to extend this support to other git providers? In the current flow, the [parsing/normalization happens here](https://github.com/vercel/api/blob/d0b5f7828a39c3ca60cc8b3796bedd9b71822fd8/services/api-deployments/src/endpoints/create-deployment.ts#L1630-L1633)